packages/hash/frontend/src/pages/index.page.tsx

```diff
@@ -1,46 +1,27 @@
 import { useEffect } from "react";
 import { useRouter } from "next/router";
 
-import { Box, Container, Typography } from "@mui/material";
-import { useLogoutFlow } from "../components/hooks/useLogoutFlow";
 import { useLoggedInUser } from "../components/hooks/useAuthenticatedUser";
 import { NextPageWithLayout } from "../shared/layout";
-import { Button } from "../shared/ui";
 
 const Page: NextPageWithLayout = () => {
   const router = useRouter();
-  const { authenticatedUser, kratosSession } = useLoggedInUser();
-
-  const { logout } = useLogoutFlow();
+  const { authenticatedUser, loading, kratosSession } = useLoggedInUser();
 
   useEffect(() => {
+    /**
+     * @todo: this check could occur in a server-side render, so that a
+     * redirect to the workspace or login page is done before rendering
+     * this empty homepage.
+     *
+     * @see https://app.asana.com/0/1203179076056209/1203451531168818/f
+     */
     if (authenticatedUser) {
       void router.push(`/${authenticatedUser.userAccountId}`);
     }
-  }, [router, authenticatedUser]);
+  }, [router, authenticatedUser, kratosSession, loading]);
```

Comment:
May be more of a dependency list question, but would these (`kratosSession, loading`) be required here?

Reply:
Fixed in https://github.com/hashintel/hash/pull/1529/commits/add5b75801745614266c96b36b39019e1549227f